### PR TITLE
[Trivial] Adding label for amount inside of tx_valid/tx_invalid.json

### PR DESCRIPTION
**Pull request by Christewart**

### CODE REVIEW

File: src/test/data/tx_invalid.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [
 ["The following are deserialized transactions which are invalid."],
 ["They are in the form"],
-["[[[prevout hash, prevout index, prevout scriptPubKey], [input 2], ...],"],
+["[[[prevout hash, prevout index, prevout scriptPubKey, amount], [input 2], ...],"],
```
Comment (jonasschnelli) [inline src/test/data/tx_invalid.json]: IMO the amount if optional, maybe we should use `(, amount)`.

**Comment by Christewart** [comment]
@jonasschnelli I changed it to `amount?`, this is a convention that is already in https://github.com/bitcoin/bitcoin/blob/b83264d9c7a8ddb79f64bd9540caddc8632ef31f/src/test/data/script_tests.json#L2

**Comment by laanwj** [comment]
LGTM, utACK 988ce2d

**Comment by kallewoof** [comment]
utACK 988ce2d

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/commit/988ce2ddef5e23da70de6ee37759a53145c6abaf

**Pull request merged**